honeybee/cli/create.py

```diff
@@ -0,0 +1,207 @@
+            for face in room[2:]:  # make all other face adiabatic
+                face.boundary_condition = ad_bc
+
+        # create the model object
+        model_id = 'Shoe_Box_Model_{}'.format(unique_id)
+        model = Model(model_id, [room], units=units,
+                      tolerance=tolerance, angle_tolerance=1)
+
+        # write the model out to the file or stdout
+        output_file.write(json.dumps(model.to_dict()))
+    except Exception as e:
+        _logger.exception('Shoe box creation failed.\n{}'.format(e))
+        sys.exit(1)
+    else:
+        sys.exit(0)
+
+
+@create.command('rectangle-plan')
+@click.argument('width', type=float)
+@click.argument('length', type=float)
+@click.argument('floor-to-floor-height', type=float)
+@click.option('--perimeter-offset', help='An optional positive number that will be used '
+              'to offset the perimeter of the footprint to create core/perimeter Rooms. '
+              'If this value is 0, no offset will occur and each floor will have one '
+              'Room', type=float, default=0, show_default=True)
+@click.option('--story-count', help='An integer for the number of stories to generate.',
+              type=int, default=1, show_default=True)
+@click.option('--orientation-angle', help='A number between 0 and 360 for the clockwise'
+              'orientation that the width of the box is facing.(0 = North, 90 = East, '
+              '180 = South, 270 = West).', type=float, default=0, show_default=True)
+@click.option('--outdoor-roof/--adiabatic-roof', help='Flag to note whether the roof '
+              'faces of the top floor should be outdoor or adiabatic.',
+              default=True, show_default=True)
+@click.option('--ground-floor/--adiabatic-floor', help='Flag to note whether the floor '
+              'faces of the bottom floor should be ground or adiabatic.',
+              default=True, show_default=True)
+@click.option('--units', help=' Text for the units system in which the model geometry '
+              'exists. Choose from (Meters, Millimeters, Feet, Inches, Centimeters).',
+              type=str, default='Meters', show_default=True)
+@click.option('--tolerance', help='The maximum difference between x, y, and z values '
+              'at which vertices are considered equivalent.',
+              type=float, default=0.01, show_default=True)
+@click.option('--output-file', help='Optional file to output the Model JSON string. '
+              'By default it will be printed out to stdout',
+              type=click.File('w'), default='-')
+def rectangle_plan(width, length, floor_to_floor_height, perimeter_offset, story_count,
+                   orientation_angle, outdoor_roof, ground_floor, units, tolerance,
+                   output_file):
+    """Create a model with a rectangular floor plan.\n
+    Note that the resulting Rooms in the model won't have any windows or solved
+    adjacencies and the edit commands should be used for this purpose.\n
```

Comment:
Hmmm. Again I think we should support such features in this command. What stops you from at least doing the solve adjacency. Or I should say in what scenario do you imagine one will not want to do the solve adjacency and how common is that?

Reply:
I don't want to create a command with 30+ inputs.  You see that the solve adjacency command already has 5 options. Moving those over to here isn't just going to create a lot of copy/pasted code.  It's going to mean that this command has almost 15 inputs.

Reply:
I'm used to Radiance so 15 options is not really something that I would mind but I can see that it can be potentially confusing for people. Let's keep them separate for now and see how it will turn out in the future. We can always add another command that does it all in one go.